src/sarif-viewer.test.ts

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import { AxeResults } from 'axe-core';
+import * as fs from 'fs';
+import { convertAxeToSarif } from './index';
+
+describe('Azure DevOps sarif viewer extension test', () => {
+    it('save sarif file to artifact to be shown by sarif viewer tab', () => {
+        const axeJSON: string = fs.readFileSync(
+            './src/test-resources/axe322-v2.dequemars-testsite.1554329251110.json',
+            'utf8',
+        );
+        const axeResultStub: AxeResults = JSON.parse(axeJSON) as AxeResults;
+        fs.mkdir('./test-results', () => {
+            fs.writeFileSync(
+                './test-results/test_result.sarif',
+                JSON.stringify(convertAxeToSarif(axeResultStub)),
```

Comment:
It would be good to explicitly specify the third options parameter as `{ encoding: 'utf8' }`, for consistency with the read operation.

Reply:
accept